src/content/api/cli.md

```diff
@@ -206,119 +251,112 @@ webpack.js index=./src/index.js index2=./src/index2.js --output-path='./dist' --
+| ----------------------- | ------------------------------------------------------------------- | ----------------------------------------------- |
+| `--bail`                | Abort the compilation on first error                                |
+| `--cache`               | Enable in memory caching [Enabled by default for watch]             | `--cache=false`                                 |
+| `--define`              | Define any free variable, see [shimming](/guides/shimming/)         | `--define process.env.NODE_ENV="'development'"` |
+| `--hot`                 | Enables [Hot Module Replacement](/concepts/hot-module-replacement/) | `--hot=true`                                    |
+| `--labeled-modules`     | Enables labeled modules [Uses LabeledModulesPlugin]                 |
+| `--live-reload`         | Enables live reloading                                              | `--live-reload=true`                            |
+| `--plugin`              | Load this [plugin](/configuration/plugins/)                         |
+| `--prefetch`            | Prefetch the particular file                                        | `--prefetch=./files.js`                         |
+| `--provide`             | Provide these modules as globals, see [shimming](/guides/shimming/) | `--provide jQuery=jquery`                       |
+| `--records-input-path`  | Path to the records file (reading)                                  |
+| `--records-output-path` | Path to the records file (writing)                                  |
+| `--records-path`        | Path to the records file                                            |
+| `--target`              | The [targeted](/configuration/target/) execution environment        | `--target='node'`                               |
+
+## Negated Flags
+
+| Shortcut   | Replaces                                                      |
+| ---------- | ------------------------------------------------------------- |
+| --no-color | Disabled any color on the console                             |
+| --no-hot   | Disabled hot reloading if you have it enabled via your config |
+| --no-stats | Disables any compilation stats emitted by webpack             |
+
+## Analyzing Bundle
+
+You can also use `webpack-bundle-analyzer` to analyze your output bundles emitted by webpack. You can use `--analyze` flag to invoke it via CLI.
+
+```sh
+webpack --analyze
+```
+
+W> Make sure you have `webpack-bundle-analyzer` installed in your project else CLI will prompt you to install it.
+
+## Aliases
+
+We've set certain aliases for commonly used command and flags
+
+| Shortcut | Replaces                                                                |
+| -------- | ----------------------------------------------------------------------- |
+| -d       | `--debug --devtool cheap-module-eval-source-map --output-pathinfo`      |
+| -p       | `--mode production`, see [building for production](/guides/production/) |
```

Comment:
I think we don't have this in v4. Can you confirm?

Reply:
yeah shortcuts need to be updated, next commit